plugins/services/src/js/pages/task-details/TaskFileViewer.js

```diff
@@ -12,7 +12,22 @@ import SearchLog from "../../components/SearchLog";
 import TaskDirectory from "../../structs/TaskDirectory";
 import TaskDirectoryActions from "../../events/TaskDirectoryActions";
 
-class TaskFileViewer extends React.Component {
+export function getNewRoute(routePath, params, path) {
+  const hasFilePathParam = routePath.indexOf(":filePath") !== -1;
```

Comment:
Not a blocker but why not `includes(":filePath")` instead `indexOf(":filePath") !== -1`?

Reply:
Just because I forgot it in the original implementation. This is just a backport 🙈 but next time I will use includes